### PR TITLE
(fix) O3-2758: changed cursor from text pointer to hand pointer.

### PR DESCRIPTION
**Pull request by subhamkumarr**
## Requirements

- [X] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [X] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [X] My work includes tests or is validated by existing tests.

## Summary
When selecting a date, the cursor is currently a text pointer, but it should be a hand pointer for a better user experience. Changed the cursor from text pointer to hand pointer.

### Before:
SS

### After: 
S1

## Related Issue
https://openmrs.atlassian.net/browse/O3-2758

### CODE REVIEW

**Comment by subhamkumarr** [comment]
@sherrif10 please review

File: packages/esm-appointments-app/src/appointments-header/appointments-header.component.tsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ const AppointmentsHeader: React.FC<AppointmentHeaderProps> = ({ title, onChange
             dateFormat="d-M-Y"
             datePickerType="single">
             <DatePickerInput
-              style={{ backgroundColor: 'transparent', border: 'none', maxWidth: '10rem' }}
+              style={{ cursor: 'pointer', backgroundColor: 'transparent', border: 'none', maxWidth: '10rem' }}
```
Comment (sherrif10) [inline packages/esm-appointments-app/src/appointments-header/appointments-header.component.tsx:45]: LGTM.

**Review by sherrif10 — APPROVED** [review]

**Review by sherrif10 — APPROVED** [review]
Thanks @subhamkumarr

**Comment by subhamkumarr** [comment]
@sherrif10 can you please merge my PR

**Review by vasharma05 — APPROVED** [review]

**Comment by subhamkumarr** [comment]
@vasharma05, will this automatically get merged?

**Comment by subhamkumarr** [comment]
Thank you @sherrif10 @vasharma05  Successfully merged my first PR 😭❤️

**Pull request merged**